tests/rbd/rbd_utils.py

```diff
@@ -89,6 +89,7 @@ def exec_cmd(self, **kw):
                 )
                 return out
 
+            err = ""
```

Comment:
we can directly assign Is it?

Reply:
When it is a long running command only out is returned, and if user gives all=True we are returning out and err variables both. In this case it will throw error that err variable is not defined. That's why i am just assigning it empty value.